### ISSUE TITLE
Suggestion/issue: The only pipeline is in the cave? Why not on the farm too?

### ISSUE THREAD

**Issue by Slayerhero90** [naelstrof/KoboldKare#129]
I was excited to mess around with the pipeline when I bought it, so I tested it on the 1 setting, got a kobold, decided I'd sell her, ran all the way back to the farm to do so, and only then discovered that the obvious pipeline platform in the farm does not also activate.

Is this intentional? If so, it is very inconvenient to run between the depths of the cave and the farm with every kobold I get from the pipeline but don't want to keep.

**Comment by naelstrof** [comment]
Yeah... The cave is due for a big refactor. It needs to be about 10% of its current size. The pipeline needs a rework too so you can't just spam it. Its the quickest way to lag a server.

My plan is to make the cave smaller, then also put a endless chasm in it so you can easily dispose of kobolds you don't want.

**Comment by hrrghdrrgh** [comment]
as opposed to a chasm, which would open more possibility for bugs with the kobolds sailing down and clipping into the ground and gradually causing lag cause they wouldn't actually despawn like you want them to, why not a red 'returns' bin that functions like the mailbox but doesn't dispense money
